.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -76,7 +76,6 @@ const uploadPipeline = (pipelineContent: string | object) => {
       GITHUB_PR_LABELS.includes('ci:all-cypress-suites')
     ) {
       pipeline.push(getPipeline('.buildkite/pipelines/pull_request/security_solution.yml'));
```

Comment:
These files now just run:
* cypress burn
* OpenAPI codegen
* Serverless OS Query tests

I don't think these steps match the `doAnyChangesMatch` call above. 

For example, changes to /^fleet_packages\.json/ probably shouldn't trigger those actions. Also, all PRs will run our tests now.

With that in mind, I think we need to re-evaluate these filters.

@xcrzx I believe you added the OpenAPI codegen step. Do you know when it needs to run?

@patrykkopycinski Should Serverless Osquery Cypress Tests run on all PRs now?

Reply:
> @xcrzx I believe you added the OpenAPI codegen step. Do you know when it needs to run?

We need to run it whenever there are changes to schema files (`*.schema.yaml`) or generated artifacts (`*.gen.ts`) in Security Solution. But as we're making the codegen script configurable, other teams might also start using it in their plugins.

Reply:
yes, all Servereless Cypre Tests should should run on all PRs